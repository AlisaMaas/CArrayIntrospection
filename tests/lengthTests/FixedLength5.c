int foo(int *a, int len) {
    int sum = 0;
    int *a_end = a + len;
    while (a != a_end) {
        sum += *a;
        a++;
   } 
   return sum;
}