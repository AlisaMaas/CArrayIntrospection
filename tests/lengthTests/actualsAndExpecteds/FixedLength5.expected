foo with argument 0 should be annotated Parameter length of 1.
